frontend/awx/views/jobs/JobOutput/JobEventEllipsis.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  border-radius: 1em;
+  background-color: var(--pf-global--BackgroundColor--light-200, white);
```

Comment:
This doesn't work in dark mode. I think this might be the wrong variable name; my dev tools say it isn't set to anything.

Reply:
What about set it to white directly? It works in dark and white mode.